index.d.ts

```diff
@@ -774,6 +774,46 @@ declare namespace WAWebJS {
     export interface PrivateChat extends Chat {
 
     }
+
+    export interface GroupChat extends Chat {
+        /** Group owner */
+        owner: any;
+        /** Date at which the group was created */
+        createdAt: Date;
+        /** Group description */
+        description: string;
+        /** Group participants */
+        participantes: Array<any>;
+        /** Adds a list of participants by ID to the group */
+        addParticipants: (participantIds: Array<string>) => Promise<void>;
```

Comment:
Returns an object, but it's a bit tricky to type because it has keys that correspond to the contact IDs that are trying to be added, indicating a status for each one that gives info about whether the user was able to be added or not.

Example:
```js
await chat.addParticipants(['1234567890@c.us'])

{
   status: 207, // (integer)
    participants: [
        '1234567890@c.us': {
            code: '200' // (string)
        }
    ],
    '1234567890@c.us': 200 // (integer)
}
```

I'm not sure how you would want to type this. Maybe `<any>` is fine for now- it might be worth to process the output at a later point to make it a bit more user-friendly, but that would introduce breaking changes for people that are relying on the current behavior. I'm open to feedback on this.

Reply:
I'm not sure if I understood well the type of the key `participants`. Would that be:

```javascript
{
    status: number;
    participants: Array<{
        [key: string]: {
           code: number
        }
    }>
} & {
    [key: string]: number;
}

//This translates to:

{
    status: 207,
    participants: [
        {
            '1111111111@c.us': {
                code: '200' 
            }
        },
        {
            '2222222222@c.us': {
                code: '200' 
            }
        }
    ],
    '1111111111@c.us': 200,
    '2222222222@c.us': 200, 
}
```

What about this one:

```javascript
{
    status: number;
    participants: {
        [key: string]: {
            code: number
        }
    }
} & {
    [key: string]: number;
}

//This translates to:

{
    status: 207,
    participants: {
        '1111111111@c.us': {
            code: '200' 
        },
        '2222222222@c.us': {
            code: '200' 
        }
    },
    '1111111111@c.us': 200,
    '2222222222@c.us': 200, 
}
```

Reply:
About processing this output, I think `participants` key is unecessary, because we can easily access its value on those properties:
```javascript
{
    ...
    '1111111111@c.us': 200,
    '2222222222@c.us': 200,
}
```

Reply:
It would be your first example (the one where participants is an array)

Reply:
done